cap/01-categorie.tex

```diff
@@ -130,7 +150,86 @@ \subsubsection*{Esercizi}
 \end{enumerate}
 
 
-\section{Isomorfismi}
+\section{Isomorfismi}\label{isomorfismi}
+
+\begin{definition}
+ Dati due oggetti $X$ e $Y$ in una categoria $\ctC$, un \emph{isomorfismo} tra $X$ e $Y$ è una coppia di morfismi $f:X\to Y$ e $g:Y\to X$ tali che $g\circ f=\id_X$ e $f\circ g=\id_Y$. 
+ $f$ è chiamato l'\emph{inverso} di $g$, e a volte si indica con $g^{-1}$, e viceversa.
+ 
+ Se esiste un isomorfismo tra $X$ e $Y$ diciamo che i due oggetti sono \emph{isomorfi}.
+\end{definition}
+
+Può essere utile un'interpretazione grafica: intuitivamente, un isomorfismo è una situazione dove l'applicazione di $f$ si può invertire o annullare tramite $g$, e viceversa, e si può sempre passare da $X$ a $Y$ senza perdere informazioni. 
+$$
```

Comment:
Ok l'interpretazione grafica; ma direi una cosa meno imprecisa: "un isomorfismo è il dato di due frecce in direzioni opposte, come in [...] con la proprietà della definizione; intuitivamente ciò significa che l'applicazione di $f$ si può invertire o annullare tramite $g$... e si può sempre passare..."